### ISSUE TITLE
9.0.0 crashing in terminal with base app, errnum 2147483651

### ISSUE THREAD

**Issue by ErraticFox** [electron/electron#23753]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 9.0.0
* **Operating System:**
  * Windows 10 (19631.1)
* **Last Known Working Electron version:**
  * 8.2.0

### Expected Behavior

Electron to start

### Actual Behavior

Crashes in terminal with error number 2147483651
### To Reproduce

I'm not quite sure what's causing this as I've cloned the `electron-quick-start` repo and before that I followed the `quick-start` document to a T and I've set up plenty of Electron projects with no problems like this in the past. Weirdly enough, it's not working with my project or a new project, but electron *is* opening in a older (only by a month or two from now) Electron project of mine. Apparently it's at 8.2.0

Changed the blank skeleton apps Electron version to 8.2.0 and it worked. This might be a problem with the Windows version I'm using (Insider Preview), but I'm confused why 8.2.0 works.

**Comment by codebytere** [comment]
We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests. This issue at present is missing a reproducible sample, without which we are forced to make assumptions that may not reflect the circumstances in which you ran your app.

We will be able to more closely look at your issue once you do so. This issue will be closed in 7 days if the above is not addressed.

**Comment by mattacton** [comment]
I tried delving into Electronjs for the first time over this past weekend and ran into this issue when trying to run the quickstart. My steps to reproduce: 

1. Have Windows 10 Insider Preview (Windows 10 Insider Preview 19631.1 (mn_release) is my current version)
1. Follow the quick start 
1. Try `npm start` 

As soon as I changed the version of Electron to 8.2.0, it began working fine.

**Comment by vishalnarwal99** [comment]
I am also having similar problem while running Electron 9.0.0.
Using Windows 10 Insider Preview 19631.1.mn_release.200514-1410
The problem occurs while running npm start resulting:

> electron-quick-start@1.0.0 start D:\docs\git repos\electron-quick-start
> electron .
npm ERR! code ELIFECYCLE
npm ERR! errno 2147483651
npm ERR! electron-quick-start@1.0.0 start: `electron .`
npm ERR! Exit status 2147483651
npm ERR!
npm ERR! Failed at the electron-quick-start@1.0.0 start script.                                                         npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Era404TM\AppData\Roaming\npm-cache\_logs\\[2020-05-27T03_13_03_929Z-debug.log](https://github.com/electron/electron/files/4686283/2020-05-27T03_13_03_929Z-debug.log)

**Comment by vishalnarwal99** [comment]
I am also having similar problem while running Electron 9.0.0.
Using Windows 10 Insider Preview 19631.1.mn_release.200514-1410
The problem occurs while running npm start resulting:
> electron-quick-start@1.0.0 start D:\docs\git repos\electron-quick-start
> electron .
npm ERR! code ELIFECYCLE
npm ERR! errno 2147483651
npm ERR! electron-quick-start@1.0.0 start: `electron .`
npm ERR! Exit status 2147483651
npm ERR!
npm ERR! Failed at the electron-quick-start@1.0.0 start script.                                                         npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Era404TM\AppData\Roaming\npm-cache\_logs\\[2020-05-27T03_13_03_929Z-debug.log](https://github.com/electron/electron/files/4686283/2020-05-27T03_13_03_929Z-debug.log)

**Comment by vishalnarwal99** [comment]
I am also having similar problem while running Electron 9.0.0.
Using Windows 10 Insider Preview 19631.1.mn_release.200514-1410
The problem occurs while running npm start resulting:
> electron-quick-start@1.0.0 start D:\docs\git repos\electron-quick-start
> electron .
npm ERR! code ELIFECYCLE
npm ERR! errno 2147483651
npm ERR! electron-quick-start@1.0.0 start: `electron .`
npm ERR! Exit status 2147483651
npm ERR!
npm ERR! Failed at the electron-quick-start@1.0.0 start script.                                                         npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\%username%\AppData\Roaming\npm-cache\_logs\\[2020-05-27T03_13_03_929Z-debug.log](https://github.com/electron/electron/files/4686316/2020-05-27T03_13_03_929Z-debug.log)

**Comment by vishalnarwal99** [comment]
I am also having similar problem while running Electron 9.0.0.
Using Windows 10 Insider Preview 19631.1.mn_release.200514-1410
The problem occurs while running npm start resulting:
> electron-quick-start@1.0.0 start D:\\docs\\git repos\\electron-quick-start
> electron .
npm ERR! code ELIFECYCLE
npm ERR! errno 2147483651
npm ERR! electron-quick-start@1.0.0 start: `electron .`
npm ERR! Exit status 2147483651
npm ERR!
npm ERR! Failed at the electron-quick-start@1.0.0 start script.                                                         npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\\Users\\%username%\\AppData\\Roaming\\npm-cache\\_logs\\[2020-05-27T03_13_03_929Z-debug.log](https://github.com/electron/electron/files/4686316/2020-05-27T03_13_03_929Z-debug.log)

**Comment by ErraticFox** [comment]
> We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests. This issue at present is missing a reproducible sample, without which we are forced to make assumptions that may not reflect the circumstances in which you ran your app.
> 
> We will be able to more closely look at your issue once you do so. This issue will be closed in 7 days if the above is not addressed.

Do I need a repo when it's now apparent it's between the OS and the Electron version?

**Comment by RainBoard01** [comment]
I have windows 10 insider preview too. Same exact error.

**Comment by craigjbass** [comment]
I also encountered this issue with the preview build of windows.

I set the `electron.exe` in `node_modules\electron\dist\electron.exe` to use compatibility mode for Windows 8 and I was able to get my development environment working.

I also noticed my packaged production electron apps encounter the same issue, the same fix works.

**Comment by ErraticFox** [comment]
@codebytere can we have this unblocked considering it's apparent to the Windows build?

**Comment by craigjbass** [comment]
I updated to Windows 10 OS Build `19640.1` last night and the issue still persists

**Comment by codebytere** [comment]
Duplicate of https://github.com/electron/electron/issues/23680

**Comment by ErraticFox** [comment]
@codebytere these do not have the same error. His last working version is 6.x.y whereas ours is 8.3.0. Also, from what I can see, it isn't the same error in the log either.

**Comment by felixrieseberg** [comment]
It's _probably_ the same thing as #23841. We'll tackle it there, just because it has a neat stack trace.

**Comment by tap6** [comment]
**I encountered a similar problem. Just after installing Fiddle, it seems to have an example when it is opened. Clicking the run button in the upper left corner will directly prompt an error. Trying to switch the Electron version is useless.**

My nodejs version is v14.19.0,
The version of electron is v16.0.2
The system is Windows 10 Home Chinese version, the version number is 20H2, and the operating system internal version is 19042.1526.

The error output seems to be the LOG in Chromium, which is line 505 of [https://chromium.googlesource.com/chromium/src/+/refs/heads/main/gin/v8_initializer.cc](url).
Attached below is a picture of when the error occurs.
!(https://github.com/tap6/pics/blob/main/Screenshot%202022-02-14%20013600.png?raw=true)
